frontend/packages/component-library/src/typography/Typography.tsx

```diff
@@ -3,7 +3,7 @@ import {memo, ReactNode, CSSProperties} from 'react';
 
 import {SemanticTag, VisualAppearance} from './types';
 
-import moduleStyles from './typography.module.scss';
+import moduleStyles from '@code-dot-org/css/typography.module.scss';
```

Comment:
Since it's .module.scss file - will it work in terms of classNames?

Reply:
It should, but I agree we don't need the `.module.scss`. I've removed it.

Reply:
Turns out we use the modularized scss because it's used in `Typography` for `visualAppearance`